apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.tsx

```diff
@@ -63,30 +65,67 @@ function getTourSteps(
 	isSiteEditor = false,
 	themeName: string | null = null
 ): WpcomStep[] {
+	const completedFlow = getQueryArg( window.location.href, 'completedFlow' );
 	const isVideoMaker = 'videomaker' === ( themeName ?? '' );
+	const isPatternAssemblerFlow = 'pattern_assembler' === completedFlow;
+	const siteEditorCourseUrl = `https://wordpress.com/home/${ window.location.hostname }?courseSlug=site-editor-quick-start`;
+	const onSiteEditorCourseLinkClick = () => {
+		recordTracksEvent( 'calypso_editor_wpcom_tour_site_editor_course_link_click', {
+			is_pattern_assembler: isPatternAssemblerFlow,
```

Comment:
For future-proof, I think we should put the name of the flow explicitly (eg: pattern\_assembler, seller...) How do you think?

Reply:
I don't think we need to put the flow name for now as it shows the congrats message only when the user goes through the pattern assembler flow. We can add the flow name when we need it in the future 🙂